controller/RestDelivery.php

```diff
@@ -225,8 +225,7 @@ public function getStatus()
             }
 
             $data = $this->getTaskLogReturnData(
-                $this->getRequestParameter(self::TASK_ID_PARAM),
-                CompileDelivery::class
```

Comment:
It it a good idea to remove this? It would change the behaviour of the endpoint. Use \oat\taoTaskQueue\controller\TaskQueueRestApi:get() instead.